src/content/release-notes.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.38.0
```

Comment:
@pchico83 Should we mention that we have removed the limits for the eventsExporter component of insights by default? That could be considered a breaking change

Also, should we mention something about the internal certificate change for the different webhooks? I guess this upgrade can generate some minimal downtime as the certificate will have to be recreated

Reply:
Re: eventExporter. This component is not documented. I will create an issue to document it, but given that the eventExporter is not documented today, I wouldn't mention the removal of the limits.
Re: internal certificate. We should mention it because it might generate issues on upgrade. We have unified all internal certificates in a single one:
https://www.okteto.com/docs/1.38/self-hosted/helm-configuration/#internalcertificate
During this upgrade, while all componets read the new certificate, there might be temporal issues in internal communication between components (a few seconds max).
I would also call out that, for customers installing Okteto with ArgoCD, to revisit our Argo CD guide to avoid issues during the upgrade:
https://www.okteto.com/docs/1.38/self-hosted/manage/argocd/